### ISSUE TITLE
How to use ethereum-package to run external EL/CL/VC nodes for the running Private testnet.

### ISSUE THREAD

**Issue by Nomad-Free-Talent** [ethpandaops/ethereum-package#795]
I started my own private Ethereum network by using the ethereum-package. I verified that all the nodes (EL/CL/VC/relay/builder) are working properly as expected.

Now, I want to expand the scale of the network and spin up additional EL/CL/VC nodes on a separate machine that can sync with the existing private network. However, I was only able to find guides and instructions related to setting up nodes for public testnets like Holesky or Sepolia.

Could you please provide guidance on:

- Configuring new external EL/CL/VC nodes to connect and sync with my existing private testnet network.
- Any specific configurations, flags, or parameters I need to be aware of for both the existing and new nodes.
- How to ensure the new nodes are properly discovered by the private network, including any necessary changes to networking settings (like static peers or bootnodes).
- Security recommendations for adding external nodes to a private testnet in a way that maintains the integrity of the private network.

Thank you in advance for your help!

**Comment by franjoespejo** [comment]
As a user I was able to do that by using static ports and the additional service `apache` which exposes the config file for external clients

**Comment by parithosh** [comment]
Yes please use static ports in order to expose your node. But I will mention that this really isn't the usecase that the ethereum-package is meant for.
